### ISSUE TITLE
[compat] webkit-text-fill-property-{002,003?} violate css-pseudo

### ISSUE THREAD

**Issue by delan** [web-platform-tests/wpt#35381]
* http://wpt.live/compat/webkit-text-fill-color-property-002.html
* http://wpt.live/compat/webkit-text-fill-color-property-003.html

The tests are currently incorrect because ‘-webkit-text-fill-color’ is not applicable to highlight pseudos like ::selection (w3c/csswg-drafts#7580). If the property was made applicable, webkit-text-fill-property-002 would still be incorrect because highlight pseudos (as specified) no longer inherit from the originating element (and only ‘color’ would affect the ::selection ‘color:currentColor’).

(@mrego)

**Comment by 1der1** [comment]
Yes, because of this issue, you can't style a proper text selection for text using the webkit-text-fill-property.
Example: The title in
https://1der1.com/
